src/blueprint.js

```diff
@@ -170,9 +170,16 @@ module.exports = {
       }) // /reduce
 
       if (outcomes.validationErrors.length) {
+        const messages = outcomes.validationErrors.map(errGroup => errGroup.err.message)
+        const error = new Error(`Invalid ${name}: ${messages.join(', ')}`)
+        error.keyErrGroup = outcomes.validationErrors
+        const invalids = outcomes.validationErrors.reduce((prev, errGroup) => ({ ...prev, [errGroup.key]: errGroup.err.message }), {})
+        error.invalids = invalids
+        const errors = outcomes.validationErrors.map(validationErr => validationErr.err)
```

Comment:
Returning just the error does not seem very helpful as there is no field association.

Reply:
My intention is to expose the app's error properties (such as `status`). @geoffgscott What are you thinking for the schema, so how mix the field association (`invalids`) with the error codes?